sdk/identity/azure-identity/tests/test_shared_cache_credential_async.py

```diff
@@ -321,10 +316,6 @@
     with pytest.raises(ClientAuthenticationError) as ex:
         await credential.get_token("scope")
 
-    # error message should mention multiple accounts and list usernames in the cache
-    assert upn_a in ex.value.message and upn_b in ex.value.message
-    assert ex.value.message.splitlines()[:-1] == MULTIPLE_ACCOUNTS.splitlines()[:-1]
```

Comment:
This test should still assert on the error message. It should fail when a bug causes the credential to raise the wrong error message.

Reply:
added as `with pytest.raises(ClientAuthenticationError, match=MULTIPLE_ACCOUNTS)`